typing/btype.mli

```diff
@@ -120,6 +121,13 @@ val proxy: type_expr -> type_expr
         (* Return the proxy representative of the type: either itself
            or a row variable *)
 
+(**** Utilities for poly types ****)
+
+(* These three functions can only be called on [Tpoly] nodes. *)
+val tpoly_is_mono : type_expr -> bool
```

Comment:
Wouldn't it be more consistent with `is_poly_Tpoly` for this to be `is_mono_Tpoly`?